ztp/argocd/deployment/resource-hook-role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: converter
+  namespace: ran-sites 
```

Comment:
use NS clusters-sub as the clusters-app https://github.com/serngawy/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/clusters-app.yaml

Reply:
fixed